src/model/assignments.model.ts

```diff
@@ -0,0 +1,49 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn, DeleteDateColumn } from 'typeorm'
+
+@Entity('assignments')
+export default class Assignment {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @Column()
```

Comment:
I think this aught to be a `@ManyToOne(() => Course, course => course.id)` at least I think that's the syntax, be sure to actually create some data in your DB to be sure this is working as expected.

Reply:
According to [this post](https://github.com/typeorm/typeorm/issues/832) we can name the column with the `@JoinColumn({ name: 'course_id' })` decorator. @jessehartloff and I were looking into this and thinking that _might_ cause it to auto query the course table whenever we pull an assignment, this'll be a good test to see if that is indeed the case. Let's hope it doesn't ([the docs](https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations) aren't super clear), so we'll wanna double check that what's happening is what we expect once this is implemented.